src/hooks/useDarkMagic.ts

```diff
@@ -37,7 +39,23 @@ const pipx = async (arg0: string | undefined) => {
     return {
       project: "pypa.github.io/pipx",
       constraint: new Range("*"),
-      shebang: ["pipx", "run"]
+      shebang: ["pipx", "run"],
+      precmd: undefined
+    }
+  }
+
+  return undefined
+}
+
+const cargo = async (arg0: string | undefined) => {
+  const res = await useFetch(`https://crates.io/api/v1/crates/${arg0}`)
```

Comment:
Is this working via coincidence? 90% of the time I am sure the executable and pkg are named the same, but it's not guaranteed right?

Reply:
Correct. We'd need some kind of massive data collection to read out the mappings.

Reply:
massive data collection was in the spec I passed to @owenjames but that can be a next step. It's great to get something already.

I think rebuilds and glibc/linux should come first but will leave that to Owen to figure out.